django_comments/views/comments.py

```diff
@@ -16,6 +17,12 @@
 from django_comments.views.utils import next_redirect, confirmation_view
 
 
+if django.VERSION < (1, 8):
+    from django.db.models import get_model
+else:
+    from django.db.models.loading import get_model
```

Comment:
isn't it now django.apps.apps.get_model?

https://docs.djangoproject.com/en/1.8/ref/applications/

One possibility: `from django.apps import apps` in 1.8, and in 1.7: `from django.db import models as apps`. Then `apps.get_model()` in the code